ui/lib/components/ColumnsSelector/index.tsx

```diff
@@ -51,54 +51,34 @@ export default function ColumnsSelector({
   }, [visibleColumnKeys, columns])
 
   const dropdownMenus = (
-    <Menu>
-      {headExtra && <Menu.Item key="head">{headExtra}</Menu.Item>}
-      {headExtra && <Menu.Divider />}
-
+    <Space direction="vertical">
       {columns
         .filter((c) => c.key !== 'dummy')
         .map((column) => (
-          <Menu.Item key={column.key}>
-            <Checkbox
-              checked={visibleKeys[column.key]}
-              onChange={(e) => {
-                onChange &&
-                  onChange({
-                    ...visibleKeys,
-                    [column.key]: e.target.checked,
-                  })
-              }}
-            >
-              {column.name}
-            </Checkbox>
-          </Menu.Item>
+          <Checkbox
```

Comment:
It can be

```
Space
  Checkbox
  Checkbox
Footer
  Divider
  Space
    Checkbox
```

Then maybe no extra styles are needed

Reply:
I tried this layout with Divider before, because all of them are inside the div with class `ant-popover-inner` which has default 16px padding, we still need to add extra style to Divider to dismiss its default padding.

Later I found the Popover native title part implement its bottom divider by border-bottom, so I just copy it...